scripts/qa_crowbarsetup.sh

```diff
@@ -1217,12 +1219,20 @@ function onadmin_crowbar_register()
 
     local adminfqdn=`crowbar machines list | grep crowbar`
     local adminip=`knife node show $adminfqdn -a crowbar.network.admin.address | awk '{print $2}'`
+
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
And if we keep it like this, please adjust the documentation in ```mkcloud``` to mention howto enable the option.